pkg/apis/psmdb/v1/psmdb_defaults.go

```diff
@@ -236,7 +237,8 @@ func (cr *PerconaServerMongoDB) CheckNSetDefaults(platform version.Platform, log
 				},
 			}
 
-			if cr.CompareVersion("1.7.0") >= 0 && !cr.Spec.UnsafeConf {
+			if (cr.CompareVersion("1.7.0") >= 0 && cr.CompareVersion("1.15.0") < 0) ||
+				cr.CompareVersion("1.15.0") >= 0 && !cr.Spec.UnsafeConf {
```

Comment:
we seem to changed the behavior, with these we'll add these flags to probe command for all clusters <1.15

Reply:
We should check for unsafe option only for clusters with >=1.15.0 version to maintain the old behavior for older cluster versions.

It seems that you checked the diff of only my changes, and not those of the entire pull request.